### ISSUE TITLE
Exception: OCA\GeoBlocker\LocalizationServices\MaxMindGeoLite2::getCountryCodeFromIPImpl(): Return value must be of type string...

### ISSUE THREAD

**Issue by Wotisrv** [HomeITAdmin/nextcloud_geoblocker#104]
Hei,

Currently installed GeoBlocker app for Nextcloud gives me this error in the log file:
`Exception: OCA\GeoBlocker\LocalizationServices\MaxMindGeoLite2::getCountryCodeFromIPImpl(): Return value must be of type string, null returned in file '/var/www/html/nextcloud/apps/geoblocker/lib/LocalizationServices/MaxMindGeoLite2.php' line 95`

It appears every time when I go into Nextcloud settings / Geoblocker. 

System: 
Fedora-Linux 36 (Server-Edition)
Nextcloud 24.0.3
PHP 8.1.8
Apache 2.4.54

**Comment by HomeITAdmin** [comment]
Which version of geoip2.phar are you using?

**Comment by Wotisrv** [comment]
2.12.2 the latest available on github.

**Comment by HomeITAdmin** [comment]
There is a good chance that it is connected to #103, although it is not 100% how. 
Unfortunately I cannot reproduce it.
When the bugfix for the other issue is out please try again. If it is still happening I have to add a check for this specific case.

**Comment by fishfish-fish** [comment]
Yet another with this issue. This time its latest Nextcloud 32.0.2:

System:
Ubuntu 22.04.5 LTS aarch64
Nextcloud: 32.0.2
GeoBlocker: 0.5.18
PHP Version: 8.3.28
Apache: 2.4.52
github.com/maxmind/GeoIP2-php: 3.3.0

In the NC - settings/admin/geoblocker with selected MaxMindGeoLite2:
`
Status of the chosen service:
Status of the service cannot be determined.
`
geoipupdate is works, latest geoip2.phar is in place with mmdb files under /var/lib/GeoIP/, and even passed internal PHP test of geoip2.phar.

In the logs - same error
`"OCA\\GeoBlocker\\LocalizationServices\\MaxMindGeoLite2::getCountryCodeFromIPImpl(): Return value must be of type string, null returned in file '/www/nextcloud/apps/geoblocker/lib/LocalizationServices/MaxMindGeoLite2.php' line 98"`

Bottom line: tried all version of geoip2.phar from 2022 till latest - no luck.
